### ISSUE TITLE
Connection to device succeeded but no datapoints found, please try again. Create a new issue and include debug logs if problem persists.

### ISSUE THREAD

**Issue by Openwrtfunboy** [rospogrigio/localtuya#717]
## Environment

- Localtuya version:  
- Last working localtuya version (if known and relevant): 
- Home Assistant Core version:  
- [] Are you using the Home Assistant Tuya Cloud component ? 
- [] Are you using the Tuya App in parallel ? 

## Steps to reproduce

1.
2. 
3.

## Configuration `configuration.yaml` or `config_flow`

```yaml

```

## Provide Home Assistant taceback/logs

```txt

```
